### ISSUE TITLE
Flocs in Tube Settler

### ISSUE THREAD

**Issue by adalian** [AguaClara/Floc_coagulant#4]
The flocs are rising to the top of the tube settler rather than entering the floc weir. This causes the effluent turbidity to be over 3 NTU. Could the source of the issue be the way I fabricated the tube settler? @monroews @lainey-reed @smp339

**Comment by monroews** [comment]
I'm wondering if a floc blanket had formed. Turbidity is always high before the floc blanket forms.

**Comment by adalian** [comment]
Yes, I noticed that that was an issue, I did not let the NTU stabilize to 100 NTU before letting the influent enter the tube settler. I added bypasses to resolve it. 
Thank you!

**Comment by monroews** [comment]
I wouldn't worry about the initial fluctuations in turbidity. That will flush through the system quickly in any case. The floc blanket will take several hours to form.

**Comment by adalian** [comment]
It seemed like there were a lot of flocs instantly because the initial NTU stayed at around 700 for about 30 minutes each time I ran an experiment. Is that okay?

**Comment by monroews** [comment]
Hmmm... The prolonged high turbidity means that something isn't working well with the PID control. One possibility is that if ProCoDA is running for a long time with the turbidity control not active, then the PID code sees a long time when the turbidity was lower than the target. This causes the I term in the PID to go to a very large value. One way to prevent this problem is likely to quit ProCoDA and then restart it and immediately start PID control.

I also wonder if the I term is too large causing it to integrate over an extended period of time that is longer than optimal.
